tests/e2e/upgrade-vertica/25-assert.yaml

```diff
@@ -0,0 +1,27 @@
+# (c) Copyright [2021] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: v-upgrade-vertica-defsc
+status:
+  replicas: 3
+---
+apiVersion: vertica.com/v1beta1
+kind: VerticaDB
+metadata:
+  name: v-upgrade-vertica
+status:
+  subclusters:
+    - installCount: 2
```

Comment:
Why are we expecting installCount to be 2?  It's a 3 node cluster.

Reply:
because when you update the crd with a wrong image, the last pod will try to update the image and fail with an ImagePullErr status so the installCount will go down to 2. That is why we delete the statefulset, to update all the other pods.

Reply:
Okay, makes sense.  Lets add a comment here so its obvious.